<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginClass</class>
 <widget class="QWidget" name="LoginClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>700</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>500</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Giris Ekrani</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(64, 80, 108);</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>271</width>
     <height>547</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="loginimg_lbl">
      <property name="minimumSize">
       <size>
        <width>269</width>
        <height>221</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>269</width>
        <height>221</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Icons/loginImg.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="username_lbl">
      <property name="minimumSize">
       <size>
        <width>269</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>269</width>
        <height>31</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;</string>
      </property>
      <property name="text">
       <string>Kullanici Adi</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="username_Line">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>31</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>18</number>
      </property>
      <property name="placeholderText">
       <string>Kullanici adi</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="pass_lbl">
      <property name="minimumSize">
       <size>
        <width>269</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>269</width>
        <height>31</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;</string>
      </property>
      <property name="text">
       <string>Sifre</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="pass_Line">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>31</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>18</number>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>Sifre</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>100</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="login_Btn">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(249, 115, 65);
font: 18pt &quot;.AppleSystemUIFont&quot;;</string>
        </property>
        <property name="text">
         <string>Giris</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../Icons/hasta.qrc"/>
 </resources>
 <connections/>
</ui>
